### ISSUE TITLE
Make selected tab listing's highlight darker

### ISSUE THREAD

**Issue by awalgarg** [Zirak/switcheroo#5]
[#eee](https://github.com/Zirak/switcheroo/commit/bdb564a33d7811a654dc8600048f5f0481f24419#diff-352dcbe9f1d38284458cc6b2e7d207eeR66) is too light, and barely a visible highlight on my monitor.

a very quick patch makes it look slightly better:

```
awal@bookstro:~/Documents/experiments/switcheroo$ git format-patch master --stdout 
From ec04bcf924e9d2d5ec13ada97489f96a7612031c Mon Sep 17 00:00:00 2001
From: awalGarg <awalgarg@gmail.com>
Date: Thu, 3 Sep 2015 22:59:47 +0530
Subject: [PATCH] make selected tab listing highlight darker

---
 src/content/style.css | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/src/content/style.css b/src/content/style.css
index 5cd482a..0088f30 100644
--- a/src/content/style.css
+++ b/src/content/style.css
@@ -70,7 +70,12 @@
 }

 .switcheroo-container .switcheroo-selected {
-    background : #eee;
+    background-color: #333;
+    color: white;
+}
+
+.switcheroo-container .switcheroo-selected .switcheroo-title {
+    color: inherit;
 }

 .switcheroo-container .switcheroo-list .switcheroo-item {
-- 
1.9.1

```

![image](https://cloud.githubusercontent.com/assets/9100409/9665777/10c6f924-5290-11e5-8fc6-14df3a6b77db.png)

sample screeny

this can be improved, ofcourse

**Comment by Zirak** [comment]
That's a tad too dark, it contrasts too much with the regular background. How about instead we make everything lighter?

![](http://i.stack.imgur.com/WNwdw.png)

**Comment by awalgarg** [comment]
LGTM :)

**Comment by Zirak** [comment]
Solved by 85dc6c1113243fe71ecb7084006c38455e067563.
